src/dune_rules/preprocessing.ml

```diff
@@ -716,6 +716,11 @@ let pp_one_module sctx ~lib_name ~scope ~preprocessor_deps
                           ; Command.Ml_kind.ppx_driver_flag ml_kind
                           ; Dep (Path.build src)
                           ; As flags
+                          ; As
+                              [ "-cookie"
+                              ; sprintf "inline-test-partition=%S"
```

Comment:
I don't think dune should be responsible for choosing the partitions.
Ppx_inline_tests should probably just get the basename from ppxlib.

Reply:
Now removed